### DISCUSSION TITLE
📲“自由软件”使用手册

### DISCUSSION THREAD

**Discussion by 1Haschwalth** [1Haschwalth/Translation#12] [Show and tell]
# 自由软件
自由软件|github项目地址|官网地址|官网下载|国内镜像网站（建议下载）
:-:|:-:|:-:|:-:|:-:
**F-Droid应用商店**|**https://github.com/f-droid/fdroidclient**|https://f-droid.org|https://f-droid.org/zh_Hans/packages/|**http://www.xfdown.com/soft/131061.html**
**Lemmy自由贴吧**|**https://github.com/LemmyNet/lemmy**|https://join-lemmy.org|https://join-lemmy.org/|**https://www.oschina.net/p/lemmy**
**Peertube自由视频平台**|**https://github.com/Chocobozzz/PeerTube**|https://joinpeertube.org/|https://github.com/Chocobozzz/PeerTube/releases|**https://gitee.com/mirrors/PeerTube**
## F-Droid应用商店
安卓客户端-自由软件存储库系统，尊重自由与隐私的应用商店。（[详见↓](https://github.com/1Haschwalth/Translation/discussions/12?sort=new#discussioncomment-7494546)）

官网地址：
- https://f-droid.org
- **github项目地址：https://github.com/f-droid**
  - **https://github.com/f-droid/fdroidclient**

下载地址：
- 官网：https://f-droid.org/zh_Hans/packages/
- **国内镜像网站（建议下载）：http://www.xfdown.com/soft/131061.html**
## Lemmy自由贴吧
联邦世界的链接聚合器和论坛（[详见↓](https://github.com/1Haschwalth/Translation/discussions/12?sort=new#discussioncomment-7498493)）

官网地址：
- https://join-lemmy.org
- **github项目地址：https://github.com/LemmyNet**
  - **https://github.com/LemmyNet/lemmy**
    - https://github.com/LemmyNet/lemmy/blob/main/readmes/README.zh.hans.md

下载地址：
- 官网：https://join-lemmy.org/
- **国内镜像网站（建议下载）：https://www.oschina.net/p/lemmy**
## Peertube自由视频平台
官网地址：
- https://joinpeertube.org/
- **github项目地址：https://github.com/Chocobozzz/PeerTube**

下载地址：
- 官网：https://github.com/Chocobozzz/PeerTube/releases
- **gitee国内镜像（建议下载）：https://gitee.com/mirrors/PeerTube**

**Comment by 1Haschwalth** [comment]
# [XMPP](https://github.com/xsf)
## [可扩展消息和在场协议](https://github.com/1Haschwalth/Translation/blob/MAIN/Archive/Backup/arXiv/Computer%20Science/XMPP/Internet%20of%20Things%20Device%20Capabilities%2C%20%20Architectures%2C%20Protocols%2C%20and%20Smart%20%20Applications%20in%20Healthcare%20Domain%3A%20A%20Review.md#c%E5%8F%AF%E6%89%A9%E5%B1%95%E7%9A%84%E4%BF%A1%E6%81%AF%E4%BC%A0%E9%80%92%E5%92%8C%E5%AD%98%E5%9C%A8%E5%8D%8F%E8%AE%AE)
可扩展消息和在场协议（XMPP）是一种确保低带宽通信和短消息传输的协议，因此非常适合视频会议、发布-订阅系统、远程呈现、多方聊天和物联网[^84]。对于即时信息应用，XMPP 适用于身份验证、安全措施、访问控制、逐跳和端到端加密，以及与各种协议的互操作性。该协议具有三种功能：客户端、服务器和网关，可促进其中任何两个角色之间的双向通信[^85]。在这种情况下，客户端通过 TCP 协议连接到服务器，并使用 XML 流标准传输数据；服务器负责连接管理和信息路由，网关则确保分布式系统之间的可靠连接。该协议灵活、简单，可在各种应用程序之间进行通信。但是，XMPP 对设备的计算能力要求高，消耗网络带宽，传输的数据类型简单，无法提供 QoS [^86]。
[^84]:E. Al-Masri et al., “Investigating Messaging Protocols for the Internet of Things (IoT),” IEEE Access, vol. 8, pp. 94880–94911, 2020.
[^85]:M. Faiqurahman, M. M. Madani, and D. R. Akbi, “Performance of XMPP-Based Gateway for IoT Device Communication Services,” J. Teknol. dan Sist. Komput., vol. 7, no. 4, pp. 127–133, Oct. 2019.
[^86]:C.-S. Cho, W.-G. Chen, C.-K. C. Liao, J.-L. Liao, F.-F. Chen, and T.-C. Shen, “Building on the Distributed Energy Resources IoT based IEC 61850 XMPP for TPC,” in 2019 IEEE International Conference on Industrial Cyber Physical Systems (ICPS), 2019, pp. 61–66.

---

>**🌏📲 https://github.com/xsf/xmpp.org/blob/master/content/getting-started.md**

您想开始使用 XMPP 聊天吗？本页面可帮助您在几分钟内[开始使用 XMPP](https://github.com/xsf/xmpp.org/blob/master/content/getting-started.md)。
## 1. 下载客户端
有[大量 XMPP 应用程序/客户端](/software/)供您选择。以下是一些比较流行的应用程序，供您参考：
### Desktop
* {{< external-link text="Beagle IM" url="https://beagle.im/" >}} (macOS)
* {{< external-link text="Converse.js" url="https://conversejs.org/" >}} (Browser)
* {{< external-link text="Dino" url="https://dino.im/" >}} (Linux)
* {{< external-link text="Gajim" url="https://gajim.org/" >}} (Linux, Windows)
* {{< external-link text="Monal" url="https://monal-im.org/" >}} (macOS)
* {{< external-link text="Swift.IM" url="https://swift.im/swift.html" >}} (Linux, macOS, Windows)
* {{< external-link text="UWPX" url="https://uwpx.org/" >}} (Windows)
### Android
* {{< external-link text="Conversations" url="https://conversations.im/" >}}
* {{< external-link text="Quicksy" url="https://quicksy.im/" >}}
* {{< external-link text="Yaxim" url="https://yax.im" >}}
### iOS
* {{< external-link text="Monal" url="https://monal-im.org/" >}}
* {{< external-link text="Siskin IM" url="https://siskin.im/" >}}
## 2. 创建账户
与电子邮件一样，您需要在服务提供商处开设账户。您可以使用[**XMPP 提供商**列表中的公共提供商](https://providers.xmpp.net)，也可以[通过自行托管服务器](https://xmpp.org/software/)成为自己的提供商。

**XMPP 提供商**在推荐提供商时会考虑多方面的因素。还有其他一些（未经整理的）提供商列表：
* [providers.xmpp.net上的经整理的XMPP服务器列表](https://providers.xmpp.net/)
* [jabber.at上的公共XMPP服务器](https://list.jabber.at)
* [404.city上的公共XMPP服务器开放列表](https://xmpp-servers.404.city)
## 3. 登录！就是这样
使用您的客户端，登录并开始添加好友，或[访问众多聊天室之一](https://search.jabber.network/)！

**Reply by 1Haschwalth** [reply]
### 隐藏IP：
1. [Conversations](https://codeberg.org/iNPUTmice/Conversations/src/branch/master/README.md)/[blabber](https://codeberg.org/kriztan/blabber.im)：右上角三点-设置-高级设置/专家设置-通过Tor连接
2. [Orbot](https://github.com/guardianproject/orbot)：配置连接方式-Snowflake-连接-选择应用-Conversations/blabber-保存→“已连接”

**Reply by 1Haschwalth** [reply]
### 其他相关文档：
- [**Jabber/XMPP中文翻译计划**](https://wiki.jabbercn.org/%E9%A6%96%E9%A1%B5)
- [《社会主义XMPP高级用户使用说明书》2024-01-05.pdf](https://github.com/1Haschwalth/Translation/files/13848215/XMPP.2024.01.05.PDF.pdf)

**Comment by 1Haschwalth** [comment]
# [F-Droid](https://github.com/f-droid)
>**https://github.com/f-droid/fdroidclient/tree/master/metadata/zh-CN**

**尊重自由与隐私的应用商店**

F-Droid 是 Android 平台上的一个自由软件应用商店。F-Droid 客户端应用使得浏览、安装和跟踪设备上的应用更新变得轻而易举。

它可以连接到任何兼容 F-Droid 的存储库。默认存储库托管在 f-droid.org 上，其中只包含真正的自由软件。

Android 本身是开放的，意思是您可以从任何地方安装 APK 应用，但是还是有很多理由使用 F-Droid 作为自由软件应用管理器：
* 在有可用更新时收到通知
* 可选择自动下载和安装更新
* 跟踪旧版本和测试版本
* 过滤与设备不兼容的应用
* 通过类别和可搜索的描述查找应用
* 访问捐赠、源代码等应用相关链接
* 通过检查存储库索引签名和 APK 哈希值来保持安全

**Comment by 1Haschwalth** [comment]
# [Lemmy](https://github.com/LemmyNet)
>**https://join-lemmy.org/docs/introduction.html**

### [1. 介绍](https://join-lemmy.org/docs/introduction.html#introduction)
Lemmy 是一个自托管的联合社交链接聚合和讨论论坛。它由许多专注于不同主题的社区组成。用户可以发布文字、链接或图片，并与他人进行讨论。投票有助于将最有趣的项目置顶。有强大的管理工具来防止垃圾邮件和巨魔。所有这些都是完全免费和开放的，不受任何公司控制。这意味着没有广告、跟踪或秘密算法。

联盟是一种去中心化的形式。每个人都可以使用多种服务，而不是单一的中央服务。

Lemmy 网站可以单独运行。就像传统网站一样，人们可以在上面注册、发布信息、上传图片和相互交流。与传统网站不同的是，Lemmy 实例可以互操作，让用户相互交流；就像你可以从 Gmail 账户向 Outlook、Fastmail、Proton Mail 或其他任何电子邮件提供商的用户发送电子邮件一样，只要你知道他们的电子邮件地址，你就可以在任何网站上使用他们的地址提及任何人或给任何人发消息。

Lemmy 使用一种标准化的开放协议来实现联合，这种协议被称为 ActivityPub。任何通过 ActivityPub 实现联盟的软件都可以与 Lemmy 进行无缝通信，就像 Lemmy 实例之间相互通信一样。

fediverse （"联合宇宙"）是所有可以通过 ActivityPub 和万维网相互通信的实例的名称。这包括所有 Lemmy 服务器，也包括其他实现：
- [Mastodon](https://joinmastodon.org/) (microblogging)
- [PeerTube](https://joinpeertube.org/) (videos)
- [Friendica](https://friendi.ca/) (multi-purpose)
- 以及许多更多！

在实际应用中：想象一下，如果你可以通过 Reddit 账户关注一个 Facebook 群组，并在不离开账户的情况下对其帖子发表评论。如果 Facebook 和 Reddit 是使用相同协议的联合服务，这将成为可能。有了 Lemmy 账户，你就可以与任何其他兼容实例进行通信，即使它不是在 Lemmy 上运行。只需软件支持相同的 ActivityPub 协议子集即可。

与专有服务不同，任何人都可以完全自由地运行、检查、检验、复制、修改、分发和重复使用 Lemmy 源代码。就像 Lemmy 用户可以选择服务提供商一样，作为个人，你也可以自由地为 Lemmy 贡献功能，或发布包含不同功能的 Lemmy 修改版。这些修改版（也称为软件分叉）必须与原版 Lemmy 项目保持相同的自由度。因为 Lemmy 是自由软件，尊重你的自由，所以个性化不仅是允许的，而且是鼓励的。

你可以在[git 存储库](https://github.com/LemmyNet/lemmy-docs)中为本文档做出贡献。

本页根据 CC BY-SA 4.0 改编自 [Mastodon 文档](https://docs.joinmastodon.org/)。
## 对于用户
### [2. 开始使用](https://join-lemmy.org/docs/users/01-getting-started.html)
### [3. 媒体](https://join-lemmy.org/docs/users/02-media.html)
### [4. 投票和排名](https://join-lemmy.org/docs/users/03-votes-and-ranking.html)
### [5. 审核](https://join-lemmy.org/docs/users/04-moderation.html)
### [6. 抗审查](https://join-lemmy.org/docs/users/05-censorship-resistance.html)
### [7. 其它特点](https://join-lemmy.org/docs/users/06-other-features.html)
### [8. Lemmy历史](https://join-lemmy.org/docs/users/07-history-of-lemmy.html)

**Comment by 1Haschwalth** [comment]
我不是专业人士，我看到一些对xmpp的批评和质疑如下：
- https://discuss.privacyguides.net/t/why-not-xmpp/13792
- https://web.archive.org/web/20211215132539/https://infosec-handbook.eu/articles/xmpp-aitm/

希望有专业人士能够对此进行探讨，或许有比XMPP更好的选择（比如**SimpleX**）。

如果我们想要【更好的元数据隐私（与谁和何时交谈），以及针对主动网络攻击者和恶意服务器的更好安全性的】的方案，或许可以看看 [**SimpleX 的白皮书**](https://github.com/simplex-chat/simplexmq/blob/stable/protocol/overview-tjr.md)和[通讯网络的类型](https://www.privacyguides.org/zh-hant/advanced/communication-network-types/)

**Comment by 1Haschwalth** [comment]
![常用开源通讯软件的隐私性、安全性评估](https://github.com/1Haschwalth/Translation/assets/135032626/a9b201a4-192d-48d9-8887-5113d708df80)

**Comment by 1Haschwalth** [comment]
## 客户端列表
- **xmpp客户端**：https://xmpp.org/software/
- **jabber客户端**：https://github.com/topics/jabber
（总结👉[点击链接查看和 Kimi 智能助手的对话](https://kimi.moonshot.cn/share/coln0tgnsmmt3vqsncgg)）
- [**几个开源XMPP Android客户端简单比较**](https://www.cnblogs.com/imparser/p/4663376.html)

**Comment by 1Haschwalth** [comment]
### 群聊bot软件：
(2023-11-27)
- https://github.com/42wim/matterbridge
- https://github.com/JFreegman/ToxBot
- https://github.com/LonamiWebs/Telethon
- https://profanity-im.github.io/

**Comment by 1Haschwalth** [comment]
### 安卓常用自由及开源软件下载列表清单
https://codeberg.org/p23tyjujukk/fdroid-use

**Comment by 1Haschwalth** [comment]
### 即时通讯软件比较表[^1]
即时通讯软件 | 类型 | 优势 | 劣势
-- | -- | -- | --
Signal | 中心化 | 可靠的加密，音频/视频通话 | 需要电话号码
Molly | Signal分支 | 提高匿名性，额外保护 | 知名度较低
Threema | 付费中心化 | 高匿名性，本地数据存储 | 付费服务，封闭代码
TwinMe | P2P去中心化 | 无需注册，无服务器 | 功能有限
Snikket (XMPP) | 开放协议 | 自己的服务器，去中心化 | 需要技术知识
Briar Project | Mesh网络 | 无互联网工作，高隐私 | 功能有限，需要设备接近
OnionShare | 通过Tor | 高匿名，文件交换 | 速度慢
Session | 去中心化 | 完全匿名，无日志 | 功能有限
SimpleX Chat | 去中心化 | 完全匿名，无元数据 | 功能有限
Cabal | 去中心化 | 完全去中心化，封闭群体 | 知名度低，功能有限
Matrix | 去中心化 | 完全去中心化 | 初始设置可能较为复杂
Tox | 去中心化 | 完全去中心化 | 设置简单
[^1]:独眼情报《暗网通讯软件的全面评测》

**Comment by takeihisa** [comment]
感谢
